internal/venice-common/src/main/proto/VeniceReadService.proto

```diff
@@ -0,0 +1,27 @@
+syntax = 'proto3';
+package com.linkedin.venice.protocols;
+
+option java_multiple_files = true;
+
+service VeniceReadService {
```

Comment:
It seems the GPRC implementation doesn't leverage request/response headers, is it expected?

Reply:
As of right now, we can use gRPC interceptors in order to inject additional data into messages as they are sent/received from the client or server. For fields that are added to headers when we perform `STORAGE` requests, we will add another field in the client/server message in order to achieve the same functionality.

Reply:
Can you cut a ticket track this? or at least leave a TODO in the code?